src/layouts/InsightsLayout/index.vue

```diff
@@ -27,7 +31,7 @@ export default {
   overflow: hidden;
 
   & > * {
-    padding: $unnnic-spacing-sm;
+    padding: 0 16px 0 16px;
```

Comment:
```suggestion
    padding: 0 $unnnic-spacing-sm;
```
- Repeating the px becomes unnecessary in this case, as the padding property supports shortcuts to use only two values (horizontal and vertical);
- Always try to use tokens from our design system to maintain security and ease if changes are necessary.